components/features/wallet/hooks/use-asset-actions.ts

```
@@ -0,0 +1,92 @@
+				return null;
+			}
+		}, []);
+
+	const [
+		{ loading: loadingSellComplete, value: sellCompleteData },
+		sellComplete,
+	] = useAsyncFn(async (body: SellAssetCompleteBody) => {
+		setError(null);
+		try {
+			const res = await sellAssetComplete(body);
+			debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)